src/Components/homepage/Homepage.js

```diff
@@ -2,34 +2,72 @@ import { useEffect, useState } from 'react'
 import Card from '../Card/Card'
 import { fetchData } from '../../Apis/Dummy-api'
 import '../Style/Homepage.css'
+import RecentlyPlayedZone from '../Recently-played-zone/Recently-played-zone'
 
 export const Homepage = () =>  {
     const [cards, setCards] = useState([])
 	const [errorMessage, setErrorMessage] = useState([])
-
+	const [recentlyPlayedCards, setRecentlyPlayedCards] = useState([])
 	const numberOfCards = 20
 
     useEffect(() => {
-        fetchData(numberOfCards).then((res) => {setCards(res)}).catch((e) => {
-            setErrorMessage(e)
+        fetchData(numberOfCards)
+        .then((res) => {setCards(res)})
+        .catch((error) => {
+            setErrorMessage(error)
             console.error('[ERROR IN HOMEPAGE COMPONENT FETCH DATA]: ' + errorMessage)
```

Comment:
i don't think this log will display the correct errorMessage, as the value of `errorMessage` is still taken from the same render, so it's not hydrated yet, this just to be checked